### DISCUSSION TITLE
👀 Active link in `yarn serve` console

### DISCUSSION THREAD

**Discussion by DevLab2425** [ProjectEvergreen/greenwood#1169] [Show and tell]
I've created a branch with an enhancement to the `yarn serve` log output that will allow CLI and IDE users (at least VS Code) to open the URL directly from the terminal window, rather than have the IDE (again, VS Code) attempt to locate the string as a local file path. See the screenshots below for a demo.

https://github.com/ProjectEvergreen/greenwood/compare/master...DevLab2425:greenwood:ShowAndTell/active-local-serve-link

**Without protocol - IDE opens file browser input**
<img width="1016" alt="Screen Shot 2023-10-17 at 16 21 55" src="https://github.com/ProjectEvergreen/greenwood/assets/8385375/290cc642-9e49-4cc0-919c-5fde14f16694">

**With protocol - browser opens URL**
<img width="1016" alt="Screen Shot 2023-10-17 at 16 22 18" src="https://github.com/ProjectEvergreen/greenwood/assets/8385375/b9a5311b-67cd-45f5-af83-d2ec937009f2">

**Comment by thescientist13** [comment]
Oh, that's great!  We might want to apply this to the dev server too.
https://github.com/ProjectEvergreen/greenwood/blob/master/packages/cli/src/commands/develop.js#L13

If you feel up for it, I'm happy to convert this into an issue.

----

On a slightly related note, there's https://github.com/ProjectEvergreen/greenwood/issues/62 which could also be interesting to look at in this space (though you are under no obligation to do it 😅 )

**Reply by DevLab2425** [reply]
@thescientist13 

> If you feel up for it, I'm happy to convert this into an issue.

Yeah, let's do that. I'll get both commands updated. 

As for #62, I took a look and think I might have an idea. Maybe a draft PR for that one?

**Reply by thescientist13** [reply]
Issue made!  - https://github.com/ProjectEvergreen/greenwood/issues/1171

@DevLab2425 
Since appending the protocol should be pretty quick, can you target the [**release/0.29.0** branch]() with this change?  (we're close to putting out another release!  👀 )
